examples/BMP388-self-test/BMP388-self-test.ino

```diff
@@ -0,0 +1,141 @@
+/**
+ * @brief BMP388 self-test using 107-Arduino-BMP388 library.
+ *
+ * according to application note https://www.bosch-sensortec.com/media/boschsensortec/downloads/application_notes_1/bst-mps-an006.pdf
+ */
+
+/**************************************************************************************
+ * INCLUDE
+ **************************************************************************************/
+
+#include <SPI.h>
+
+#include <ArduinoBMP388.h>
+
+/**************************************************************************************
+ * CONSTANTS
+ **************************************************************************************/
+
+static int const BMP388_CS_PIN  = 2;
+static int const BMP388_INT_PIN = 6;
+
+/**************************************************************************************
+ * FUNCTION DECLARATION
+ **************************************************************************************/
+
+void    spi_select     ();
+void    spi_deselect   ();
+uint8_t spi_transfer   (uint8_t const);
+void    onExternalEvent();
+void    onSensorData   (double const pressure_hpa, double const temperature_deg);
+
+/**************************************************************************************
+ * GLOBAL VARIABLES
+ **************************************************************************************/
+
+ArduinoBMP388 bmp388(spi_select,
+                     spi_deselect,
+                     spi_transfer,
+                     onSensorData);
+volatile bool measurement_done;
```

Comment:
I suggest immediate initialisation with an initial value, as usually all variables should be assigned a value upon declaration (good defensive coding practive), e.g.
```C++
volatile bool measurement_done = false;
volatile double measurement_pressure_hpa = 0.0;
...
```